docs/tutorials/custom_puzzle_secure.md

```diff
@@ -0,0 +1,82 @@
+---
+id: custom_puzzle_secure
+title: How to secure coin with a custom puzzle
+sidebar_label: Secure coin with a custom puzzle
+---
```

Comment:
I think you should change the name of this one because a password locked coin is actually very insecure: https://chialisp.com/docs/security#password-locked-coin-security .  I think it's a helpful tutorial but it shouldn't be labelled secure or people may try to actually spend to this and that would be very bad.

Reply:
I've changed it to "lock", which I meant with "secure". I hope that is better phrasing.